br.com.barros.DevWidgets.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "br.com.barros.DevWidgets",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "devwidgets",
+    "separate-locales": false,
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--share=network"
+    ],
+    "modules": [
+      {
+        "name": "devwidgets",
+        "buildsystem": "simple",
+        "only-arches": [
+          "x86_64"
```

Comment:
this doesn't do what you think it does. You are still trying to build an aarch64 flatpak from this.

Reply:
What is the correct form? My intent is to build for x64